### DISCUSSION TITLE
JEI plugin confusion

### DISCUSSION THREAD

**Discussion by jgrable634** [shedaniel/RoughlyEnoughItems#961] [Q & A] [answered]
Is there any proper guide for setting up REI with JEI plugins? I feel like a idiot but I don't want to break something, so any help would be greatly appreciated

**Comment by shedaniel** [comment, marked as answer]
Yes, you will need https://www.curseforge.com/minecraft/mc-mods/roughly-enough-items-hacks

**Reply by jgrable634** [reply]
So I just install that and then JEI plugins right? No other steps are necessary right?

To be clear I am using a fabric modpack

**Reply by shedaniel** [reply]
No, REI does not support JEI plugins on Fabric, I am also not aware of any mods implementing the JEI api at the moment.

**Reply by jgrable634** [reply]
Alright, thank you for the info

**Reply by CodexAdrian** [reply]
I currently use JEI for plugins on Fabric. Are there any plans to bring the wrapper to fabric?

**Reply by shedaniel** [reply]
> I currently use JEI for plugins on Fabric. Are there any plans to bring the wrapper to fabric?

No, a single mod using JEI for plugins isn't that significant for a compatibility layer, compatibility layers have serious performance implications, stability issues and generally not fun to develop for
